File: tests/integration_tests/tutorial_fr/programs/maze_fr.py
def tourne_a_droite():
    repeat 3:
        tourne_a_gauche()

def suit_le_mur():
    if rien_a_droite():
        tourne_a_droite()
        avance()
    elif rien_devant():
        avance()
    else:
        tourne_a_gauche()

while not au_but():
    suit_le_mur()

File: tests/integration_tests/tutorial_fr/programs/maze_fr.js
//
function droite() {
    tourne_a_gauche();
    tourne_à_gauche();  // test accent
    tourne_a_gauche();
}

function suit_le_mur () {
    if (rien_a_droite()) {
        droite();
        avance();
    } else if (rien_devant()) {
        avance();
    } else {
        tourne_a_gauche();
    }
}


while (!au_but()) {
    suit_le_mur();
}
